packages/fxa-auth-server/lib/routes/linked-accounts.ts

```diff
@@ -151,6 +152,8 @@ export class LinkedAccountHandler {
             );
             rawIdToken = res.data['id_token'];
             idToken = jose.decodeJwt(rawIdToken);
+            this.log.info("linked_account.apple_id_token", idToken);
```

Comment:
I don't think we need to log out the `id_token` here since that could possible expose some user data.